src/content/en/updates/2019/05/desynchronized.md

```diff
@@ -0,0 +1,189 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The desynchronized hint invokes bypasses the DOM to eliminate the latency caused by using the renderer compositor queue.
+
+{# wf_updated_on: 2019-05-02 #}
+{# wf_published_on: 2019-05-02 #}
+{# wf_tags: chrome75 #}
+{# wf_featured_image: /web/updates/images/2019/05/latency.png #}
+{# wf_featured_snippet: The desynchronized hint invokes bypasses the DOM to eliminate the latency caused by using the renderer compositor queue. #}
```

Comment:
This description needs more detail, I have zero idea what it means, this is your hook to get me interested and read the article.